share-on-mastodon.js

```diff
@@ -1,46 +1,53 @@
 // Get the page title to use if a title isn't set.
 const docTitle = document.getElementsByTagName("title")[0].innerHTML;
-const template =  document.createElement('template');
+const template = document.createElement("template");
 template.innerHTML = `
 <style>
   button:focus,
-  input:focus {
+  input:focus-visible {
     outline: var(--mastodon-focus-ring, 2px solid #3B82F6);
     outline-offset: .12rem;
   }
 
   .btn-container {
     display: inline-block;
   }
+
   .btn {
     background-color: var(--mastodon-btn-bg, #563ACC);
     border: var(--mastodon-btn-border, 1px solid #2F0C7A);
     border-radius: var(--mastodon-btn-rad, 6px);
     color: var(--mastodon-btn-txt, #fff);
     cursor: pointer;
     display: inline-flex;
+    align-items: center;
     font-weight: bold;
     font-size: 1rem;
+    line-height: 1;
     gap: .5em;
-    padding: .5em 1em;
-  }
+    padding: .75em 1em;
 
-  .btn:hover,
-  .btn:focus {
-    filter: brightness(120%);
-  }
+    &:hover,
+    &:focus-visible {
```

Comment:
Are you planning to use LESS or SASS? Correct me if I'm wrong, but in plain CSS, there is no `&` operator, right?

Reply:
There is indeed an `&` operator in vanilla css nowadays [mdn link](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting)

Reply:
Oohh nice!! I didn't know vanilla CSS nesting exists now 🎉👏🏻  Wow this is a real game changer 👍🏻 It also seems [well supported](https://caniuse.com/css-nesting) except for some mobile browsers!